images/macos/provision/core/azcopy.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash -e -o pipefail
 
+source ~/utils/utils.sh
+
 AZCOPY_DOWNLOAD_URL="https://aka.ms/downloadazcopy-v10-mac"
 
-wget -O "$HOME/azcopy.zip" "$AZCOPY_DOWNLOAD_URL"
+download_with_retries $AZCOPY_DOWNLOAD_URL "." "azcopy.zip"
```

Comment:
Is it better to download to `/tmp` like you have done in other places?

Reply:
Done